frontend/src/pages/Projects/Project/ProjectDetails/Deliverables/tableConfig.tsx

```diff
@@ -0,0 +1,84 @@
+import { GridColDef } from "@mui/x-data-grid";
+import { dateFormatter } from "utils/formatDate";
+import percentFormatter from "utils/formatPercent";
+
+export const tableConfig = () => {
+  const defaultFlex = 4;
+  const smallFlex = 3;
+  const lineBreakStyle = { lineHeight: "1.5em" };
+
+  const lineBreak = (l1: string, l2: string) => (
```

Comment:
could this be utilized somewhere else, something like `lineBreak` seems like it is a reusable component, however it doesn't look very re-usable here.

Reply:
done.